cmd/sample/main.go

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "log"
+
+func main() {
+	log.Printf("Hello, World!")
```

Comment:
This looks like an unnecessary piece of code to maintain. Can you explain why you added it ?

Reply:
```
Unsigned images are fine in namespaces not subject to the webhook,
Unsigned images are rejected in namespaces subject to the webhook,
Signed images are fine in namespaces subject to the webhook.
```

Couldn't we build and publish to this registry running alongside the our cosign image without being signed ? So we can test it using our own docker image. 

On the other hand, if this code is only used for testing purposes. Would it be a better choice to move it to the `test` directory ? wdyt ?

Reply:
Yeah, that's what this does.  `ko publish` this test image to the local registry, so we don't need to rely on a prebuilt image.  I'd be happy to move this to a more clearly named directory, I just didn't see a precedent to follow yet.